src/ConfCrypt/Commands.hs

```diff
@@ -52,14 +53,26 @@ class Monad m => Command a m where
     evaluate :: a -> m [T.Text]
 
 -- | Read and return the full contents of an encrypted file. Provides support for using a local RSA key or an externl KMS service
-data ReadConfCrypt = ReadConfCrypt
+data ReadConfCrypt = ReadConfCrypt {rTemplate :: Maybe T.Text}
+    deriving (Eq, Read, Show, Generic)
+
 instance (Monad m, MonadDecrypt (ConfCryptM m key) key) => Command ReadConfCrypt (ConfCryptM m key) where
-    evaluate _ = do
+    evaluate (ReadConfCrypt template) = do
         (ccFile, ctx) <- ask
         let params = parameters ccFile
-        transformed <- mapM (\p -> decryptedParam  p <$> decryptValue ctx (paramValue p)) params
-        processReadLines transformed ccFile
+        case template of
```

Comment:
I don't think I completely understand what's going on here, but it smells funny.

If I was adding such feature, I would design it in such a way that "no format" is a special case of format, and would be handled in `renderTemplate`, which would take `Maybe Text` instead of `Text`.
I hesitate to state this unequivocally, because it doesn't seem clear-cut: the two branches seem to be doing significantly different things, and I don't understand why.

Reply:
The existing behavior was that `confcrypt read` would write out in the same syntax as the original confcrypt file, with the fields decrypted, including comments, spacing, etc. The new behavior I added is fundamentally different: rather than operating on the lines in the file, it operates on the data structure the file represents, and renders it into the user's desired shape. The case ended up here because this is where the original rendering logic was.

I'm open to refactoring this further, but I'm not sure exactly how to approach it.

Reply:
I'd actually prefer if the entire suite of template logic were in the CLI rather than library. I'm fine going with this for now, but I'd prefer to further separate the UI from the library (which really means changing the `Command` api)